gateway/README.md

```diff
@@ -0,0 +1,39 @@
+# javascript.client
```

Comment:
I don't intend to link subs to parents, no. They have to click the parent to see this sub.